### ISSUE TITLE
Shortcut for opening link in incognito window

### ISSUE THREAD

**Issue by tkumse** [philc/vimium#1841]
Many users use incognito mode every day and it might be a good feature to make it possible to do this through Vimium as well.

Is this possible?
What combination of keys would be best suited for this feature?

**Comment by smblott-github** [comment]
Thanks, @tkumse.

I'm not sure this warrants a new command (or new vomnibar modes) since `Ctrl-Shift-N` is available.

**Comment by tarek2** [comment]
I second this feature, it will be very helpful to have it in Vimium.

**Comment by mrmr1993** [comment]
Thirded, this would be nice to have.

`yf` (copy link) `<c-N>` (open new incognito tab) `<c-v>` (paste) `<ret>` also works, but it's clunky.

Phil's idea in #1312 might be a nice way to expose this and other link hints commands without falling foul of #1269, if that was something we wanted to do.

**Comment by nfischer** [comment]
If the user wants to open an incognito window, `Ctrl-Shift-N` seems good enough. As I see it, the best Vimium could do would be a two-key shortcut instead of the default three-key shortcut. Not a huge advantage.

Now, if you're talking about opening *a link* in incognito mode (like @mrmr1993 pointed out), that makes more sense. I'm not sure what the best key-combo would be, so I agree that the idea in #1312 might be a good way to go.

**Comment by smblott-github** [comment]
Agreed, @nfischer.  And #1312 is a nice way of side-stepping the proliferation of Vomnibar modes.

**Comment by smblott-github** [comment]
This is in #1948.

**Comment by Beasta** [comment]
For sites like medium.com, I'm constantly yanking the URL, opening an incognito, and pasting into the incognito. Would love to have an means of opening the current URL in incognito.

**Comment by snprajwal** [comment]
LinkHints.activateModeToOpenIncognito is available and can be mapped for this

**Comment by Beasta** [comment]
🔥  for anyone interested, i used @snprajwal 's advice above and created a link for capital I under custom key mappings

`map I LinkHints.activateModeToOpenIncognito`

**Comment by snprajwal** [comment]
I think we can close this issue, given that the feature has been available for quite some time now.
